buildmaster/E155_NC_9GEV_EP_G1F1RATIO/metadata.yaml

```
@@ -0,0 +1,64 @@
+# Generalia
+setname: "E155_NC_9GEV_EP_G1F1RATIO"
+
+version: 1
+version_comment: "Initial implementation"
+
+# References
+arXiv:
+   url: "https://arxiv.org/pdf/hep-ph/0007248.pdf"
+iNSPIRE:
+   url: "https://inspirehep.net/literature/530798"
+hepdata:
+   url: "https://www.hepdata.net/record/ins530798"
+   version: 1
+
+nnpdf_metadata:
+   nnpdf31_process: "DIS"
+   experiment: "E155"
+
+implemented_observables:
+  - observable_name: "NC_EP_G1F1RATIO"
+    observable: {description: "E155 DIS scattering experiment polarized/unpolarized structure function fraction", label: $\frac{g_1}{F_1}$, units: " " }
+    ndata: 24
+    tables: [1,2,3,4,5,6,7,8,9,10,11]
+    npoints: [1,1,1,2,2,3,3,3,3,3,2] # number of datapoints in each table
+
+#    ALL the following must be checked carefully
+#    plot_x: sqrttau
+#    figure_by: 
+#      - y
+#    kinematic_coverage:
+#      y: y
+#      sqrttau: sqrttau
+    dataset_label: "E155"
+    y_label: '$g_1(x, Q^2) / F_1(x, Q^2)$'
+    result_transform: qbinEMC
+    x_scale: log
+    y_scale: log
+    line_by:
+      - Q2
+    figure_by:
+      - k2bins6
+    normalize:
+  y_scale: linear
+      line_by: []
+    figure_by:
+            - k2bins6
+            - Q2
```

As per [this guideline](https://github.com/NNPDF/nnpdf/blob/2e39e52258a4b31169a879fedb40513e18697313/doc/sphinx/source/data/new-commondata.rst):
```suggestion
    # Plotting information
    plotting:
        dataset_label: "E155 (g1/F1)"
        plot_x: x
        y_label: '$g_1(x, Q^2) / F_1(x, Q^2)$'
        figure_by:
            - Q2
    kinematic_coverage: [x, y, Q2]
```